items.json

```diff
@@ -18614,6 +18614,39 @@
     },
     "weapons": []
   },
+  {
+    "id": "crpg_military_cleaver_5_t4",
+    "name": "Soldier's Cleaver",
+    "culture": "Neutral",
+    "type": "OneHandedWeapon",
+    "price": 7445,
+    "weight": 2.35,
+    "tier": 9.959684,
+    "requirement": 0,
+    "flags": [],
+    "weapons": [
+      {
+        "class": "OneHandedSword",
+        "itemUsage": "onehanded_block_shield_swing",
+        "accuracy": 0,
+        "missileSpeed": 0,
+        "stackAmount": 0,
+        "length": 100,
+        "balance": 0.47,
+        "handling": 80,
+        "bodyArmor": 2,
+        "flags": [
+          "MeleeWeapon"

```

Comment:
did you generate a thumbnail to test if there were gaps?

Reply:
@namidaka where do they get generated by default after clicking the "export thumbnails" button ingame?

Reply:
@namidaka seems to look good: 
![crpg_military_cleaver_5_t4](https://user-images.githubusercontent.com/118079593/213500878-621f550a-49a8-490c-bb0a-60011b4be03c.png)